docs/products/m3db/howto/grafana.rst

```diff
@@ -0,0 +1,37 @@
+Visualise M3DB Data with Grafana
+================================
+
+Since M3DB is best for time series data, many individual metrics, then it's nicer to visualise the data than try to view it in a table or log. Luckily, Aiven can set up the Grafana and the integration between the two services for you.
+
+Integrate M3DB and Grafana
+--------------------------
+
+1. On the service overview page for your M3DB service, go to "Manage Integrations" and choose the "Dashboard" option.
+
+2. Choose either a new or existing service.
+
+   - A new service will ask you to confirm the cloud, region and plan to use. You should also give your service a name. The service overview page shows the nodes rebuilding, and then indicates when they are ready.
```

Comment:
"select" instead of "confirm"

Reply:
Should we state something about selecting the same cloud provider and region for minimal latency?

Reply:
In general, the howto content types should have as little extra explanation as possible. Maybe we should have an explainer for cloud regions or something though if you think this is important and not already well understood.